docs/src/Button.doc.js

```diff
@@ -281,38 +284,52 @@ card(
     <Combination
       id="color"
       name="Color"
-      color={[
-        'gray',
-        'red',
-        'blue',
-        'white',
-        'transparent',
-        'transparentWhiteText',
-      ]}
+      color={['gray', 'red', 'blue', 'white', 'transparent']}
     >
-      {(props, i) => {
-        let color;
-        if (props.color === 'transparentWhiteText') {
-          color = 'darkWash';
-        }
-        if (props.color === 'transparent') {
-          color = 'lightWash';
-        }
-        return (
-          <Box color={color}>
-            <Button
-              id={`example-${i}`}
-              onChange={() => {}}
-              {...props}
-              text="Button"
-            />
-          </Box>
-        );
-      }}
+      {(props, i) => (
+        <Button
+          id={`example-${i}`}
+          onChange={() => {}}
+          {...props}
+          text="Button"
+        />
+      )}
     </Combination>
   </>
 );
 
+card(
+  <Example
+    name="Color: transparent"
+    id="transparent"
+    defaultCode={`function Example() {
+  const [selected, setSelected] = React.useState(false);
+  return (
+    <Stack gap={1}>
+      <Button text="Toggle Text Color" onClick={() => setSelected(!selected)} inline selected={selected}/>
+      <Box column={3}>
+        <Image
+          alt="Tropic greens: The taste of Petrol and Porcelain | Interior design, Vintage Sets and Unique Pieces agave"
+          color="rgb(231, 186, 176)"
+          naturalHeight={751}
+          naturalWidth={564}
+          src="https://i.ibb.co/7bQQYkX/stock2.jpg"
+        >
+          <Box padding={2}>
+          <Button color={selected ? 'transparentWhiteText' : 'transparent'} iconEnd="add-pin" text="Save this image" inline />
```

Comment:
@AlbertCarreras could we
* Fix the indentation on this example?
* Remove the selected toggle?
* Use a darker background, to make the example accessible.

Something like this

```
function Example() {
  return (
    <Box width={300}>
      <Image
        alt="Tropic greens: The taste of Petrol and Porcelain | Interior design, Vintage Sets and Unique Pieces agave"
        color="rgb(231, 186, 176)"
        naturalHeight={751}
        naturalWidth={564}
        src="https://i.ibb.co/7bQQYkX/stock2.jpg"
      >
        <Box dangerouslySetInlineStyle={{ __style: { background: 'rgba(0, 0, 0, 0.3)' } }} height="100%">
          <Box padding={2}>
            <Button color="transparentWhiteText" iconEnd="add-pin" text="Save this image" inline />
          </Box>
        </Box>
      </Image>
    </Box>
  )
};
```

![image](https://user-images.githubusercontent.com/127199/92664839-ebf82080-f2b9-11ea-9fd3-408c4f4da2c5.png)

Reply:
NOT DONE